### PR TITLE
ECL-149 : Added return journey

### PR DESCRIPTION
**Pull request by jamesvarghese701**

### CODE REVIEW

File: src/test/resources/journeys.conf [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
-      home-page,
-      post-vat-return-period,
-      get-turnover-page
+      ecl-registration-login
     ]
   }
 
+   ecl-returns = {
+      description = "Returns login for Economic Crime Levy"
+      load = 9.1
```
Comment (peter-hazell) [inline src/test/resources/journeys.conf:28]: this load is extremely high - can we lower it for now to something like 0.3 - in line with https://confluence.tools.tax.service.gov.uk/display/ECL/04.05.01+Load+testing+estimates

File: src/test/resources/services.conf [hunk 2]
```diff
@@ -19,7 +30,7 @@
 
 
 services {
-  example-frontend{
+  economic-crime-levy-status-frontend{
```
Comment (peter-hazell) [inline src/test/resources/services.conf:33]: is this meant to be `returns` or `registration`? what is `status`?

File: src/test/resources/journeys.conf (hunk 1 above)
Comment (jamesvarghese701) [reply src/test/resources/journeys.conf]: changed to 0.3 now

File: src/test/resources/services.conf (hunk 2 above)
Comment (jamesvarghese701) [reply src/test/resources/services.conf]: added return and registration services now

File: src/test/resources/services.conf [hunk 3]
```diff
@@ -19,9 +30,14 @@
 
 
 services {
-  example-frontend{
+  economic-crime-levy-registration{
```
Comment (peter-hazell) [inline src/test/resources/services.conf:33]:
```suggestion
  economic-crime-levy-registration-frontend{
```

File: src/test/resources/services.conf [hunk 4]
```diff
@@ -19,9 +30,14 @@
...
     protocol = https
     host = www.staging.tax.service.gov.uk
     port = 443
   }
+  economic-crime-levy-returns{
```
Comment (peter-hazell) [inline src/test/resources/services.conf:38]:
```suggestion
  economic-crime-levy-returns-frontend{
```

File: src/test/resources/services.conf (hunk 2 above)
Comment (peter-hazell) [reply src/test/resources/services.conf]: pls keep `frontend` appended to the end, as we have backend services which we shouldn't get them mixed up with

File: src/test/resources/services.conf (hunk 3 above)
Comment (jamesvarghese701) [reply src/test/resources/services.conf]: appended now

File: src/test/resources/services.conf (hunk 4 above)
Comment (jamesvarghese701) [reply src/test/resources/services.conf]: appended now

**Review by peter-hazell — APPROVED** [review]

**Pull request merged**